themes/default/content/templates/container-service/aws/index.md

```diff
@@ -0,0 +1,133 @@
+
+The template requires no additional configuration. Once the new project is created, you can deploy it immediately with [`pulumi up`]({{< relref "/docs/reference/cli/pulumi_up" >}}):
+
+```bash
+$ pulumi up
+```
+
+When the deployment completes, Pulumi exports the following [stack output]({{< relref "/docs/intro/concepts/stack#outputs" >}}) values:
+
+url
+: The HTTP URL for the container's endpoint.
+
+Output values like these are useful in many ways, most commonly as inputs for other stacks or related cloud resources. The computed `url`, for example, can be used from the command line to open the newly deployed container service in your favorite web browser:
+
+```bash
+$ open $(pulumi stack output url)
+```
+
+## Customizing the project
+
+Projects created with the Container Service template expose the following [configuration]({{< relref "/docs/intro/concepts/config" >}}) settings:
+
+container_port
+: Specifies the port mapping for the container. Defaults to port 80.
+
+cpu
+: Specifies the amount of CPU to use with each task or each container within a task. Defaults to 512.
+
+memory
+: Specifies the amount of memory to use with each task or each container within a task. Defaults to 128.
+
+image
+: Specifies the location of the Dockerfile used to build the container image that is run. Defaults to the Dockerfile in the `app` folder.
+
+All of these settings are optional and may be adjusted either by editing the stack configuration file directly (by default, `Pulumi.dev.yaml`) or by changing their values with [`pulumi config set`]({{< relref "/docs/reference/cli/pulumi_config_set" >}}) as shown below.
+
+### Using your own container image
+
+If you already have a container image you'd like to build your container service with, you can do so either by replacing the Dockerfile in the `app` folder or by configuring the stack to point to another folder on your computer with the `image` setting:
+
+```bash
+$ pulumi config set path ../my-existing-image/build
+$ pulumi up
+```
+
+### Cleaning up
+
+You can cleanly destroy the stack and all of its infrastructure with [`pulumi destroy`]({{< relref "/docs/reference/cli/pulumi_destroy" >}}):
+
+```bash
+$ pulumi destroy
+```
+
+## Learn more
+
+Congratulations! You're now well on your way to managing a production-grade container service on AWS with Pulumi --- and there's lots more you can do from here:
+
+* Discover more architecture templates as they're available in [Templates &rarr;]({{< relref "/templates" >}})
+* Dive into the AWS package by exploring the [API docs in the Registry &rarr;]({{< relref "/registry/packages/aws" >}})
+* Dive into the AWSx (Crosswalk) package by exploring the [API docs in the Registry &rarr;]({{< relref "/registry/packages/awsx" >}})
+* Expand your understanding of how Pulumi works in [Architecture &amp; Concepts &rarr;]({{< relref "/docs/intro/concepts" >}})
+* Read up on the latest new features [in the Pulumi Blog &rarr;]({{< relref "/blog/tag/containers" >}})
```

Comment:
My comments on serverless apply here. Don't say as they're available. You should point to the Learn repo instead of docs. Then the Pulumi Blog relref thing doesn't work for tags like the serverless one, so you should fix it like Christian did for that other page. 

I would also merge the lines of AWS and AWSx and just hyperlink the name of the package